datacube/model/schema/ingestor-config-type-schema.yaml

```diff
@@ -1,4 +1,9 @@
-"$schema": "http://json-schema.org/draft-04/schema#"
```

Comment:
this is probably esential but got removed

Reply:
Weird.